### PR TITLE
#18563 feedback bundle resource

### PR DESCRIPTION
**Pull request by erickgonzalez**
Merged Postman Tests into one file.
Added new option to the WebResource `requireLicense` and test.
Remove the unused code of the BundleResource.

### CODE REVIEW

**Review by jdotcms — APPROVED** [review]

**Review by dsilvam — APPROVED** [review]

File: dotCMS/src/main/java/com/dotcms/rest/WebResource.java [hunk 1]
```diff
@@ -760,6 +763,7 @@ private UsernamePassword(String username, String password) {
         private String[] requiredPortlet = null;
         private final Set<String> requiredRolesSet = new HashSet<>();
         private AnonymousAccess anonAccess=AnonymousAccess.NONE;
+        private boolean requireLicense = false;
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/rest/WebResource.java:766]: Codacy found an issue: [Avoid using redundant field initializer for 'requireLicense'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5883793)

File: dotCMS/src/main/java/com/dotcms/rest/WebResource.java [hunk 2]
```diff
@@ -861,7 +870,12 @@ public InitDataObject init() {
                 Logger.debug(InitBuilder.class,
                         () -> request.getRequestURI()+" calling webResource.init(..) with AnonymousAccess set to `" + anonAccess + "`");
             }
-            
+
+            if(requireLicense){
+                if (LicenseUtil.getLevel() < LicenseLevel.STANDARD.level) {
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/rest/WebResource.java:875]: Codacy found an issue: [These nested if statements could be combined](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5883793)

File: dotCMS/src/integration-test/java/com/dotcms/rest/WebResourceIntegrationTest.java [hunk 3]
```diff
@@ -342,4 +345,12 @@ public void disallow_anon_when_required_frontend_but_not_backend() throws Except
 
   }
 
+  @Test(expected = InvalidLicenseException.class)
+  public void webResource_NoLicense_RequiredLicense_returnInvalidLicenseException() throws Exception {
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/WebResourceIntegrationTest.java:349]: Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5883793)

File: dotCMS/src/main/java/com/dotcms/rest/WebResource.java (hunk 1 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/rest/WebResource.java:766]: Codacy found an issue: [Field requireLicense has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5883793)

**Pull request merged**